libopae/CMakeLists.txt

```diff
@@ -41,12 +41,19 @@ target_include_directories(opae-c PUBLIC
   PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}
   PUBLIC ${libjson-c_INCLUDE_DIRS})
 
-add_library(opae-c-ase SHARED init_ase.c)
+set(SRC_ASE
+    pluginmgr.c
+    api-shell.c
```

Comment:
Why are we adding pluginmgr.c, api-shell.c, and props.c in this library? Aren't those compiled into libopae-c.so?

Reply:
@r-rojo This is to resolve the concerns Michael had at https://github.com/OPAE/opae-sdk/pull/1265.